src/helpers/helpers.canvas.js

```diff
@@ -26,6 +26,27 @@ export function setBorderStyle(ctx, options) {
   }
 }
 
+/**
+ * Apply shadow options to the canvas context before drawing a shape
+ * @param {CanvasRenderingContext2D} ctx - chart canvas context
+ * @param {Object} options - options with shadow configuration
+ */
+export function setShadowStyle(ctx, options) {
+  ctx.shadowColor = options.shadowColor;
+  ctx.shadowBlur = options.shadowBlur;
+  ctx.shadowOffsetX = options.shadowOffsetX;
+  ctx.shadowOffsetY = options.shadowOffsetY;
+}
+
+/**
+ * Resets shadow options to the canvas context before stroking,
+ * in order to manage teh shadow at "fill" level
+ * @param {CanvasRenderingContext2D} ctx - chart canvas context
+ */
+export function resetShadow(ctx) {
+  ctx.shadowColor = 'transparent';
+}
```

Comment:
Either all shadown propertys should be reset as the descriptions and fucntion name suggest or the description and name should be changed to something like `setShadowColorTransparent` and the description something like `set shadow color to transparent before stroking in order to manage the shadow at "fill" level`.

Also small nitpick `the` on line 43 between `manage` and `shadow` has a typo

Reply:
I have to disagree slightly. I would not create a function called `setShadowColorToTransparent` in any case. Maybe a better name could be `disableShadow`.

The description however is off, because in this function you can not know if its called between fill and stroke. I'd remove the description and maybe add some comment in the code using this before stroke.

Reply:
:+1: